src/commands/publish.ts

```diff
@@ -575,6 +589,10 @@ export async function publishMain(argv: PublishOptions): Promise<any> {
       argv.noMerge,
       argv.keepBranch
     );
+    if (!isDryRun()) {
+      // intentionally DO NOT await unlinking
```

Comment:
Can you explain why we don't want to await here?

Reply:
We do not want to delay any further steps to wait for the IO of a file removal that doesn't matter anymore. Also, if the removal fails for some reason `await` raises an exception, blocking further steps, stalling the publish (whereas it actually succeeded).